docs/src/IconButton.doc.js

```diff
@@ -84,6 +88,23 @@ card(
   />
 );
 
+card(
+  <Example
+    name="Disabled"
+    description={`Just like any button component, \`IconButton\` can render in a disabled state`}
+    defaultCode={`
+<IconButton
+  accessibilityLabel="Love"
+  bgColor="white"
+  disabled={true}
```

Comment:
Don't we just use `disabled` instead of `disabled={true}`?

Reply:
good catch, changed!